### PR TITLE
Make a request for data

### PR DESCRIPTION
**Pull request by wademauger**
Would it make sense to pull this into the main repo?

### CODE REVIEW

File: app/js/components/tags-page.js [hunk 1]
```diff
@@ -7,10 +7,42 @@
...
+          reject('an error has occured');
+        }
+      };
+
+      xhr.send();
+    });
+
+    tagsRequest.then(tags => this.setState({...this.state, tags}));
+  }
+
```
Comment (kristenmills) [inline app/js/components/tags-page.js:45]: https://github.com/rit-sse/api-client

Comment (bbesmanoff) [inline app/js/components/tags-page.js:45]: Wanted to hold off until data was there...

-----Original Message-----

Comment (kristenmills) [inline app/js/components/tags-page.js:45]: Run the API locally

Comment (kristenmills) [inline app/js/components/tags-page.js:45]: You shouldn't really be testing your application using the production API anyway.

**Pull request closed without merging**